content/docs/next-release/deployment.md

```diff
@@ -2,6 +2,9 @@
 title: Deployment
 weight: 7
 children:
+- title: Operator
```

Comment:
Should this be called Kubernetes Operator?

Reply:
Also, I don't think it worked correctly, the new page doesn't show in preview.

https://deploy-preview-258--jaegertracing.netlify.com/docs/next-release/deployment/

Reply:
The content can be viewed here: https://deploy-preview-258--jaegertracing.netlify.com/docs/next-release/operator/

Can't see why it would not show as a build page of Deployment. Any idea @lucperkins?

Reply:
"Operator for Kubernetes" -- Kubernetes Operator is ambiguous, IMO.